### ISSUE TITLE
Glossary: Unicode definition is either informal or (possibly) incorrect

### ISSUE THREAD

**Issue by ojeda** [rust-lang/fls#156]
> Unicode is the colloquial name for the ISO/IEC 10646:2017 Universal Coded Character Set standard.

If the Glossary is defining terms, it should just say what Unicode standard it is referring to, without referring to the "colloquial name" (i.e. unnecessary / too informal).

Otherwise, if the Glossary is not defining the term, and instead is giving some extra information about what "Unicode" typically is, then it is likely people refer colloquially to the Consortium's Unicode Standard instead, and not the ISO/IEC International Standard, for instance. So more clarifications would be needed.

**Comment by PLeVasseur** [comment]
@kirtchev-adacore -- do you know if this is still relevant?

I see mention of definition of `Unicode` [here](https://rust-lang.github.io/fls/glossary.html#unicode).

Was the issue related to whether this is a precise enough definition?

**Comment by kirtchev-adacore** [comment]
> Was the issue related to whether this is a precise enough definition?

Correct.

Looking at [Wikipedia](https://en.wikipedia.org/wiki/Unicode) reveals that an alias of "Unicode" is "Universal Coded Character Set (UCS)ISO/IEC 10646", which is what we defined "Unicode" to be.

@ojeda It seems to me that "Consortium's Unicode Standard" and "ISO/IEC 10646" are the same thing, no?

**Comment by ojeda** [comment]
No, my understanding is that there are two standards: the Unicode Standard and the ISO/IEC one.

https://www.unicode.org/faq/unicode_iso

**Comment by kirtchev-adacore** [comment]
OK. @PLeVasseur this issue is still relevant then.

**Comment by ojeda** [comment]
I sent a quick PR with a suggestion for both issues: https://github.com/rust-lang/fls/pull/617
